doubles/class_double.py

```diff
@@ -13,3 +34,27 @@ class ClassDouble(InstanceDouble):
     """
 
     is_class = True
+
+    def __init__(self, path):
+        super(ClassDouble, self).__init__(path)
+        self._doubles_target = patch_class(self._doubles_target)
+        self._target = Target(self._doubles_target)
+
+    def __call__(self, *args, **kwargs):
+        """
+        represents __new__ for ``ClassDouble`` instances.
```

Comment:
First line of a docstring should a one-liner. http://www.python.org/dev/peps/pep-0257/#multi-line-docstrings

Nit: docstring should prescribe the function or method's effect as a command ("Return this"), not as a description ("Returns that") :) http://www.python.org/dev/peps/pep-0257/